docker/k8s/vtctlclient/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
 
-COPY --from=k8s /vt/bin/vtctlclient /usr/bin/
+COPY --from=lite /vt/bin/vtctlclient /usr/bin/
```

Comment:
Want to point out (but it might not be worth doing anything about; and changing it would be a breaking change) that `vtctlclient` is put into `/usr/bin`, but `vtexplain` is put into `/vt/bin`. It's inconsistent that they don't both go into one or the other, or both go into both.

```sh
vitess ((HEAD detached at origin/release-9.0)) $ git grep "COPY.*/usr/bin"
docker/k8s/orchestrator/Dockerfile:33:COPY --from=k8s /vt/bin/vtctlclient /usr/bin/
docker/k8s/vtctlclient/Dockerfile:28:COPY --from=k8s /vt/bin/vtctlclient /usr/bin/
docker/orchestrator/Dockerfile:31:COPY vtctlclient /usr/bin/vtctlclient
docker/orchestrator/Dockerfile:32:COPY orchestrator /usr/bin/orchestrator
```

I guess it makes some sense that `orchestrator` (which I guess was for the old Helm chart) and `vtctlclient` don't go into `/vt/bin`, since the former isn't a Vitess component, and the latter is a client component, not part of the Vitess service. But by that same heuristic, `vtexplain` also wouldn't go into `/vt/bin`. 🤷‍♀️  Doesn't matter that much, figured I'd point it out.

```sh
vitess ((HEAD detached at origin/release-9.0)) $ git grep "COPY.*/vt/bin"
docker/k8s/Dockerfile:44:COPY --from=base /vt/bin/mysqlctld /vt/bin/
docker/k8s/Dockerfile:45:COPY --from=base /vt/bin/vtctld /vt/bin/
docker/k8s/Dockerfile:46:COPY --from=base /vt/bin/vtctl /vt/bin/
docker/k8s/Dockerfile:47:COPY --from=base /vt/bin/vtctlclient /vt/bin/
docker/k8s/Dockerfile:48:COPY --from=base /vt/bin/vtgate /vt/bin/
docker/k8s/Dockerfile:49:COPY --from=base /vt/bin/vttablet /vt/bin/
docker/k8s/Dockerfile:50:COPY --from=base /vt/bin/vtworker /vt/bin/
docker/k8s/Dockerfile:51:COPY --from=base /vt/bin/vtbackup /vt/bin/
docker/k8s/mysqlctld/Dockerfile:37:COPY --from=k8s /vt/bin/mysqlctld /vt/bin/
docker/k8s/orchestrator/Dockerfile:33:COPY --from=k8s /vt/bin/vtctlclient /usr/bin/
docker/k8s/vtbackup/Dockerfile:29:COPY --from=k8s /vt/bin/vtbackup /vt/bin/
docker/k8s/vtctl/Dockerfile:31:COPY --from=k8s /vt/bin/vtctl /vt/bin/
docker/k8s/vtctlclient/Dockerfile:28:COPY --from=k8s /vt/bin/vtctlclient /usr/bin/
docker/k8s/vtctld/Dockerfile:29:COPY --from=k8s /vt/bin/vtctld /vt/bin/
docker/k8s/vtexplain/Dockerfile:26:COPY --from=base /vt/bin/vtexplain /vt/bin/
docker/k8s/vtgate/Dockerfile:31:COPY --from=k8s /vt/bin/vtgate /vt/bin/
docker/k8s/vttablet/Dockerfile:37:COPY --from=k8s /vt/bin/vttablet /vt/bin/
docker/k8s/vttablet/Dockerfile:38:COPY --from=k8s /vt/bin/vtctlclient /vt/bin/
docker/k8s/vtworker/Dockerfile:31:COPY --from=k8s /vt/bin/vtworker /vt/bin/
docker/mini/Dockerfile:46:COPY --from=base /vt/bin/vtctl /vt/bin/
docker/mini/Dockerfile:47:COPY --from=base /vt/bin/mysqlctl /vt/bin/
```

Reply:
You are right that it would be a breaking change :(
I have anyway included it in #7433 so that we can discuss whether or not to do anything.